src/javascript/expression.rs

```diff
@@ -144,6 +146,36 @@ impl<'module> Generator<'module> {
         })
     }
 
+    fn call<'a>(&mut self, fun: &'a TypedExpr, arguments: &'a [CallArg<TypedExpr>]) -> Output<'a> {
+        let fun = self.not_in_tail_position(|gen| gen.expression(fun))?;
+        let arguments = self.not_in_tail_position(|gen| {
+            call_arguments(
+                arguments
+                    .iter()
+                    .map(|element| gen.wrap_expression(&element.value)),
+            )
+        })?;
+        Ok(docvec![fun, arguments])
+    }
+
+    fn fun<'a>(&mut self, arguments: &'a [TypedArg], body: &'a TypedExpr) -> Output<'a> {
+        let tail = self.tail_position;
+        self.tail_position = true;
+        let result = self.expression(body);
+        self.tail_position = tail;
+        Ok(docvec!(
+            docvec!(
+                fun_args(arguments),
+                " => {",
+                break_("", " "),
+                result?,
+                break_("", " ")
```

Comment:
It needs to be inside the group but not inside the nest, otherwise the `}` has one level too much indentation

Reply:
Thanks for the patience, this formatting is causing a lot of head scratching.

Reply:
I totally understand, it took me forever to learn all the tricks here. It's not obvious at all